driver-core/src/test/java/com/datastax/driver/core/MaterializedViewMetadataTest.java

```diff
@@ -75,27 +81,33 @@ public void should_create_view_metadata() {
         assertThat(mv.getColumns().get(3)).isNotNull().hasName("score").isClusteringColumn().hasClusteringOrder(DESC);
         assertThat(mv.getColumns().get(4)).isNotNull().hasName("user").isClusteringColumn();
         assertThat(mv.getColumns().get(5)).isNotNull().hasName("day").isClusteringColumn();
-
-        assertThat(mv.asCQLQuery(false)).contains(createMV);
-
+        //whereClause currently puts quoted identifiers around column names so this assertion would fail, see:
```

Comment:
I re-enabled the assertion, knowing that it will probably fail until Sylvain's suggestion is implemented.